bindings/python/tools/python/ostk/astrodynamics/utilities.py

```
@@ -19,7 +22,144 @@
+                dv_y=float(dv[1]),
+                dv_z=float(dv[2]),
+            )
+        )
+
+    return residuals
+
+
+def compute_residuals_for_orbit(
+    orbit: Orbit,
+    reference_states: list[State],
+    local_orbital_frame_factory_or_frame: LocalOrbitalFrameFactory | Frame = Frame.GCRF(),
```

Ruff (0.14.5)

Do not perform function call `Frame.GCRF` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)